### DISCUSSION TITLE
[TEMP CHECK] Introducing “Web3 UX” - A set of tools that simplify the development and use of Dapps.

### DISCUSSION THREAD

**Discussion by Christopher-I** [akash-network/community#563] [Governance Proposals]
## Summary
This proposal seeks to provide Akash's web3 developers with a suite of streamlined tools, featuring a user-friendly white-label wallet solution for seamless user onboarding, automated gas fee management, transaction notifications, and analytics without collecting user data, all aimed at enhancing the development and user experience of decentralized applications (dApps).

## Motivation
After dedicating over 7 years to navigating the complexities of the web3 landscape, particularly focusing on optimizing user experiences (UX) for both centralized and decentralized applications (Dapps), I've identified recurring challenges faced by virtually all Dapps. These challenges encompass facilitating smooth user transitions from web2 to web3, accelerating developer onboarding from web2 to web3 while maintaining productivity, addressing the intricacies of gas fee management through abstraction or innovative solutions like a user-refillable gas tank, ensuring robust seed phrase management and recovery mechanisms, delivering reliable transaction notifications via email, text, or webhooks, offering optional KYC integrations, and providing analytics without compromising user privacy.
Motivated by these widespread challenges, I am driven to develop a comprehensive solution within a unified platform, akin to the seamless experiences offered by web2. My strategy involves initially targeting the most pressing pain points experienced by platforms like Akash and gradually expanding the feature set to encompass additional UX enhancements.
Drawing from my extensive experience as a founding engineer at both BlockFi and Parallel Finance, as well as my tenure as a smart contract engineer at Ocean Protocol, I bring a wealth of insights and expertise to the table. Furthermore, my recent acceptance into Techstars Web 2024 underscores my commitment to innovation and entrepreneurial excellence.
I am eager to collaborate with the Akash team to gain deeper insights into their specific pain points and user needs. By working together closely, I am confident that we can develop impactful solutions that elevate the overall web3 experience for all stakeholders.

## Next Steps
- Gather feedback from the community.
- Figure out the greatest UX pain point that I can help solve for the community.
- Narrow down the first feature to build and open a grant proposal.

## About Me
Web3UX Website: https://www.uxweb3.com/
Personal Website: https://www.chrisigbojekwe.com/
Linkedin: https://www.linkedin.com/in/chris-i/
Github: https://github.com/Christopher-I
Twitter: https://twitter.com/chris_igbojekwe

**Comment by brewsterdrinkwater** [comment]
@Christopher-I, 

Thank you for opening up this discussion. I believe the discussion is mislabeled, as I do not see anything related to a governance proposal right now. Sounds like the goal of the discussion is to get a temp check on whether your services are needed on Akash. 

Akash Network is open source, so contributions are welcomed by ALL that want to participate. To organize the community, there is a special interest group and working group structure, where community members can collaborate and work in parrelel on a number of inititives across the entire Akash Network. 

The Akash calendar for live meetings can be found here: https://akash.network/development/welcome/#calendar

The Akash Network discord is for builders, and there are many channels where constant communication is happening between live meetings. I would recommend you join the Akash Discord, if you have not already. There you can connect with other builders, and discuss ideas with active community members. Pain points and ways to support will identify themselves quickly. 

Based upon your experience, I see a few places where you can get involved. Sig-clients features teams and community members who are building solutions on top of Akash. Many of these solutions are focused on improving the UX for users trying to use the Akash Network. I think you have nailed many of the challenges on the UX front, especially in terms of web2 -> web3 audiences. Sig-design is another special interest group that focuses on UX, front end design, and branding. 

My name is Tyler, and I am one of the admins, so please "@" me, if you need.

**Comment by Christopher-I** [comment]
Hey Tyler! Thanks for the detailed reply. Sorry about the mislabeling, I am
still new here and trying to figure out how grants/proposals are
checked/passed and yes the goal of my message was to get a temp check on
whether my services are needed on Akash.

I just joined the discord and will try to connect with other builders and
see how I can help. Also appreciate the help pointing me in the right
direction(starting point) in regards to the sig-clients/sign-design groups.
I will start there and if any questions come up, I will reach out.

On Mon, Apr 29, 2024 at 10:48 AM Drinkwater ***@***.***>
wrote:

> @Christopher-I <https://github.com/Christopher-I>,
>
> Thank you for opening up this discussion. I believe the discussion is
> mislabeled, as I do not see anything related to a governance proposal right
> now. Sounds like the goal of the discussion is to get a temp check on
> whether your services are needed on Akash.
>
> Akash Network is open source, so contributions are welcomed by ALL that
> want to participate. To organize the community, there is a special interest
> group and working group structure, where community members can collaborate
> and work in parrelel on a number of inititives across the entire Akash
> Network.
>
> The Akash calendar for live meetings can be found here:
> https://akash.network/development/welcome/#calendar
>
> The Akash Network discord is for builders, and there are many channels
> where constant communication is happening between live meetings. I would
> recommend you join the Akash Discord, if you have not already. There you
> can connect with other builders, and discuss ideas with active community
> members. Pain points and ways to support will identify themselves quickly.
>
> Based upon your experience, I see a few places where you can get involved.
> Sig-clients features teams and community members who are building solutions
> on top of Akash. Many of these solutions are focused on improving the UX
> for users trying to use the Akash Network. I think you have nailed many of
> the challenges on the UX front, especially in terms of web2 -> web3
> audiences. Sig-design is another special interest group that focuses on UX,
> front end design, and branding.
>
> My name is Tyler, and I am one of the admins, so please "@" me, if you
> need.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/akash-network/discussions/563#discussioncomment-9265661>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKQUGSA36U62FAVF4PKS2JTY7Z2UBAVCNFSM6AAAAABG5FK5RWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TENRVGY3DC>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

*Chris Igbojekwe*

Web3UX Website: https://www.uxweb3.com/

Personal Website: https://www.chrisigbojekwe.com/

Linkedin: https://www.linkedin.com/in/chris-i/
Github: https://github.com/Christopher-I
Twitter: https://twitter.com/chris_igbojekwe
